lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -454,23 +478,236 @@ end
 
 --------------------------------------------------------------------------------
 
+e2function void bone:boneManipulate(vector pos, angle rot, isFrozen, gravity, collision)
+	ent, index = boneVerify(self, this)
+	if not ValidAction(self, ent, "manipulate", this, index) then return end
+	
+	setPos(this, pos)
+	setAng(this, rot)
+	
+	this:EnableMotion(isFrozen == 0)
+	this:EnableGravity(gravity ~= 0)
+	this:EnableCollisions(collision ~= 0)
+	
+	ent:PhysWake()
+end
+
+e2function void bone:boneFreeze(isFrozen)
+	if not boneVerify(self, this) then return end
+	this:EnableMotion( isFrozen == 0 )
+	this:Wake()
+end
+
+__e2setcost(30)
+
+e2function void bone:setCollisions(enable)
+	if not boneVerify(self, this) then return end
+	this:EnableCollisions(enable ~= 0)
+	this:Wake()
+end
+
+e2function void bone:setDrag( number drag )
+	if not boneVerify(self, this) then return end
+	this:EnableDrag( drag ~= 0 )
+end
+
+e2function void bone:setInertia( vector inertia )
+	if not boneVerify(self, this) then return end
+	if Vector( inertia[1], inertia[2], inertia[3] ):IsZero() then return end
```

Comment:
The rest of propcore hasn't been updated to reflect this yet, but vectors are `Vector` userdata now. So you can replace this with just `inertia:IsZero()`, and use `inertia` below.